helm-chart/templates/_helpers.tpl

```diff
@@ -0,0 +1,68 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "kubeshark.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "kubeshark.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "kubeshark.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "kubeshark.labels" -}}
+helm.sh/chart: {{ include "kubeshark.chart" . }}
+{{ include "kubeshark.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- with .Values.additionalLabels }}
+{{ toYaml . }}
+{{- end }}
+{{- if .Values.tap.labels }}
+{{- toYaml .Values.tap.labels}}
+{{- end }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "kubeshark.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "kubeshark.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
```suggestion
app.kubernetes.io/instance: {{ eq .Release.Name "release-name" | ternary "kubeshark" .Release.Name }}
```

Wouldn't it be better if I was something like this because when I do `helm template .` I see the label: `app.kubernetes.io/instance: release-name`

Reply:
You see `release-name` because you installed it as `helm install release-name .` And That's what we expect when we install a helm chart. If you install cilium as `helm install release-name cilium/cilium` it will create every resource and label with a `release-name` on it. That's one reason why we use labels to filter resources, as [by default](https://helm.sh/docs/chart_best_practices/labels/#standard-labels), the `app.kubernetes.io/instance` label should be `{{ .Release.Name }}`.

Reply:
@Ziul we have a Makefile rule named `generate-manifests` in here: https://github.com/kubeshark/kubeshark/blob/65bda4e84432a1f8306d75f9abddc51b09e55acb/Makefile#L75-L76
Which runs `helm template ./helm-chart` command. So, we would want to replace the default name `release-name` with `kubeshark`.